exercises/kindergarten-garden/description.md

```diff
@@ -23,8 +23,8 @@ There are 12 children in the class:
 - Eve, Fred, Ginny, Harriet,
 - Ileana, Joseph, Kincaid, and Larry.
 
-Each child gets 4 cups, two on each row. The children are assigned
-cups in alphabetical order.
+Each child gets 4 cups, two on each row. Their teacher assigns cups to 
+the children alphabetically by their names.
```

Comment:
This works.

Reply:
I didn't go with my first draft of the new sentence:
```text
The evil nun assigned cups to the children alphabetically by their names.
```
Then I would just have to replace teacher everywhere.  :tongue: